### PR TITLE
update CASE REVIEW #1745243

### PR DESCRIPTION
**Pull request by v-abiss**

### CODE REVIEW

**Comment by PRMerger18** [comment]
@v-abiss : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/apps/cd/deploy-aks.md [hunk 1]
```diff
@@ -217,4 +217,10 @@ You're now ready to create a release, which means to start the process of runnin
...
 
-* [Set up multi-stage release](../../release/define-multistage-release-process.md)
\ No newline at end of file
+* [Set up multi-stage release](../../release/define-multistage-release-process.md)
+
+## FAQs
+
+1. I'm unable to deploy the artifacts to AKS cluster after enabling the Private Links for ACR.
+
+In such cases, create new Virtual Machine Scale sets and add a load balancer IP which is public. Ensure that this public IP is mentioned in the selected networks of the Azure Container Registry. Add the Azure Container Registry and public IP details in ***/etc/hosts*** file of the Virtual Machine Scale sets.
```
Comment (azooinmyluggage) [inline docs/pipelines/apps/cd/deploy-aks.md:226]: Why is a VMSS required? Is public IP not enough? I am not an networking expert so someone who knows about Azure networking should take a look.

Comment (v-abiss) [reply docs/pipelines/apps/cd/deploy-aks.md:226]: VMSS was being used when this issue was encountered. Therefore, I specifically mentioned about VMSS. However, public IP alone is good enough. I'll test it out and correct it.

**Comment by ktoliver** [comment]
This commit was moved to a new PR in the private repo, where it will be reviewed and merged. We'll close this PR. Thanks.

**Pull request closed without merging**